packages/playground/src/weblets/tf_contracts_list.vue

```diff
@@ -250,22 +234,16 @@
 </template>
 
 <script lang="ts" setup>
-import type { ContractsOverdue, GridClient } from "@threefold/grid_client";
+import type { Consumption, ContractsOverdue, GridClient } from "@threefold/grid_client";
 import { type Contract, ContractState, NodeStatus, SortByContracts, SortOrder } from "@threefold/gridproxy_client";
 import { DeploymentKeyDeletionError } from "@threefold/types";
 import { computed, defineComponent, onMounted, type Ref, ref } from "vue";
 
 import ContractsTable from "@/components/contracts_list/contracts_table.vue";
 import { useProfileManagerController } from "@/components/profile_manager_controller.vue";
 import { useProfileManager } from "@/stores/profile_manager";
-import type { VDataTableHeader } from "@/types";
-import {
-  type ContractsTableType,
-  ContractType,
-  getNodeInfo,
-  normalizeContract,
```

Comment:
i don't understand why we remove the import and reimplement the function again her e